docs/events.md

```diff
@@ -4,63 +4,60 @@ linkTitle: "Events"
 weight: 2
 ---
 -->
-# Events
+# Events in Tekton
 
-Tekton runtime resources, specifically `TaskRuns` and `PipelineRuns`,
-emit events when they are executed, so that users can monitor their lifecycle
-and react to it.
+`TaskRuns` and `PipelineRuns` emit [Kubernetes events](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#event-v1-core)
+during their execution. This allows you to monitor and react to what's happening during execution by
+retrieving those events using the `kubectl describe` command. Tekton can also emit [CloudEvents](https://github.com/cloudevents/spec).
 
-Tekton emits [kubernetes events](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#event-v1-core), that can be retrieve from the resource via `kubectl describe [resource]`.
+Tekton emits [Kubernetes events](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#event-v1-core). You that can be retrieve from the resource via `kubectl describe [resource]`.
```

Comment:
nit: `You that can be retrieve` -> `You can retrieve these` (or something to that effect)

Reply:
Oy, I shouldn't be doing rewrites late in the day. Fixed!